pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/options/impl/SkipDefaultCheckout.java

```diff
@@ -30,16 +30,18 @@
 import org.jenkinsci.plugins.pipeline.modeldefinition.options.DeclarativeOptionDescriptor;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import javax.annotation.Nullable;
+
 public class SkipDefaultCheckout extends DeclarativeOption {
-    boolean skipDefaultCheckout;
+    private Boolean skipDefaultCheckout;
```

Comment:
Isn't `private` the default?  
I don't know the style guide for jenkinsci, but it didn't have `private` before.